src/storage/hummock_sdk/src/time_travel.rs

```diff
@@ -41,8 +44,13 @@ pub fn refill_version(
             .chain(level.levels.iter_mut())
     }) {
         refill_level(level, sst_id_to_info);
+        level
```

Comment:
Can we apply the `table_id` filter when we `refill_level`?

Reply:
We have to get the true `SstableInfo` from `HashMap<HummockSstableId, SstableInfo>` to tell whether the sst can be omitted. So filtering out by table_id in inner calls like `refill_level` can only enhance performance by a assignment of `SstableInfo`, which I think is not significant.